doc/modules/classes.rst

```diff
@@ -1133,10 +1133,6 @@ See the :ref:`metrics` section of the user guide for further details.
    metrics.pairwise.polynomial_kernel
    metrics.pairwise.rbf_kernel
    metrics.pairwise.sigmoid_kernel
-   metrics.pairwise.paired_euclidean_distances
-   metrics.pairwise.paired_manhattan_distances
-   metrics.pairwise.paired_cosine_distances
-   metrics.pairwise.paired_distances
```

Comment:
Can these be moved to https://github.com/scikit-learn/scikit-learn/blob/93f6ce994b92513a35f2ee417c77bc21d4d9fdc0/doc/modules/classes.rst?plain=1#L1903-L1904?